### PR TITLE
docs: add note about current GeoPoint type support

### PR DESCRIPTION
**Pull request by kursattokpinar**
Signed-off-by: kursattokpinar <kursattokpinar@gmail.com>

GeoPoint implementation is not completed yet. It'd be better if we add a note to Loopback Types page.
See [Cannot start the application. Error: Unsupported type: geopoint](https://github.com/strongloop/loopback-next/issues/1981)"

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/LoopBack-types.md [hunk 1]
```diff
@@ -130,6 +130,9 @@ value is provided.
  The type name is case-insensitive; so for example you can use either \"Number\" or \"number\".
 " %}
 
+{% include note.html content="The support for `GeoPoint` is not finished yet.
+Related GH issue: [Cannot start the application. Error: Unsupported type: geopoint](https://github.com/strongloop/loopback-next/issues/1981)" %}
```
Comment (achrinza) [inline docs/site/LoopBack-types.md:134]:
```suggestion
{% include note.html content="`GeoPoint` is not supported. See GitHub issue
[#1981](https://github.com/strongloop/loopback-next/issues/1981)" %}
```

Let's consider rephrasing it to be more concise.

**Review by achrinza — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a minor request on the phrasing 👇

File: docs/site/LoopBack-types.md (hunk 1 above)
Comment (kursattokpinar) [reply docs/site/LoopBack-types.md:134]: I was guessing that it's under development and will be available soon. If it is not supported at all, wouldn't it be better if we remove all the documentation about the GeoPoint type.

Comment (achrinza) [reply docs/site/LoopBack-types.md:134]: Apologies for the misunderstanding.

The way I interpret it is: under development ≠ supported

For something to be supported, it needs to be developed. From the perspective of the user, the feature is not supported, regardless if it's under development or not.

The link to the open GitHub issue acts as a signal the feature is under consideration or development.

Hence the suggestion to just say "not supported" just to be concise.

Comment (kursattokpinar) [reply docs/site/LoopBack-types.md]: Alright, thank you for making it clearer. I am committing your suggestion.
Very minor change but my very first contribution to an oss ever! Hope I can contribute more! 👍

**Review by achrinza — APPROVED** [review]
Hi @kursattokpinar, I've went ahead and tidied the Pull Request's commit history. Could you please confirm that everything (such as DCO sign-off and changes) is correct?

**Comment by kursattokpinar** [comment]
Hi @achrinza, yes both sign-off and changes seems correct, thank you for your help.
I couldn't handle signing off by amending the commits made over GitHub GUI. Thanks 👍

**Comment by achrinza** [comment]
Thanks for the Pull Request! It's been merged to master.

**Pull request merged**